WordPress/Classes/ViewRelated/Plugins/CollectionViewContainerRow.swift

```diff
@@ -0,0 +1,178 @@
+/// A `ImmuTableRow` that contains a `UICollectionView`.
+/// Currently the `CollectionViewContainerCell` is fairly specific to needs of Plugin Directory,
+/// but it should be possible to make it more generic in the future if there's a need to — it was deliberately
+/// not made as generic as possible, in case it won't be used.
+
+private let CellReuseIdentifier = "CollectionViewContainerRowReuseIdentifier"
```

Comment:
Although this is a private constant, since it refers to `CollectionViewContainerRow`, how about moving it as a fileprivate constant inside the struct?